lib/ClangImporter/ClangImporter.cpp

```diff
@@ -2217,19 +2217,20 @@ bool PlatformAvailability::treatDeprecatedAsUnavailable(
 }
 
 ClangImporter::Implementation::Implementation(
-    ASTContext &ctx,
```

Comment:
This change initializes the `DiagnosticWalker`. The remaining changes are just ClangFormat.